.github/workflows/deploy_mainnet.yml

```
@@ -0,0 +1,64 @@
+name: Deploy Explorer Indexer
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_tag:
+        description: 'The Release tag Version'
+        required: true
+        default: 'stable' 
+        type: string
+
+env:
+  AWS_REGION: '${{ secrets.AWS_REGION }}'
+  ENVIRONMENT: MAINNET
+  AWS_MAINNET: '${{ secrets.AWS_MAINNET }}'
+  REGISTRY: 'ghcr.io'
+
+jobs:
+  deploy:
+   name: deploy
+   runs-on: ubuntu-latest
+
+   permissions:
+     contents: read
+     id-token: write
+     actions: write
+
+   steps:
+     - name: checkout the source code
+       uses: actions/checkout@v3
+
+     - name: checkout ecs repo
+       uses: actions/checkout@v3
+       with:
+         repository: sygmaprotocol/devops
+         token: ${{ secrets.GHCR_TOKEN }}
+         ref: main
+
+     - name: render jinja2 templates to task definition json files
+       uses: cuchi/jinja2-action@v1.2.0
+       with:
+         template: 'sygma-explorer/ecs/task_definition-explorer-indexer-${{ env.ENVIRONMENT }}.j2'
+         output_file: 'sygma-explorer/ecs/task_definition-explorer-indexer-${{ env.ENVIRONMENT }}.json'
+         data_format: json
+         variables: |
+           awsAccountId=${{ env.AWS_MAINNET }}
+           awsRegion=${{ env.AWS_REGION }}
+           DB_USERNAME=${{ secrets.MAIN_USERNAME }}
+           DB_PASSWROD=${{ secrets.MAIN_PASSWORD }}
+
+     - name: Configure AWS Credentials
+       uses: aws-actions/configure-aws-credentials@v1
+       with:
+         role-to-assume: arn:aws:iam::${{ env.AWS_MAINNET }}:role/github-actions-${{ env.ENVIRONMENT }}-chainbridge
```

```suggestion
         role-to-assume: arn:aws:iam::${{ env.AWS_MAINNET }}:role/github-actions-${{ env.ENVIRONMENT }}-chainbridge
```

Shouldn't this be sygma?

---

Same for api.